docs(cli): deprecate Kilo Console (#12701)
Kilo-Org/kilocode: packages/kilo-docs/pages/contributing/architecture/index.md

## Repository boundaries

{% callout type="warning" title="Kilo Console is deprecated" %}

> References to Kilo Console and `kilo console` on the architecture pages describe a deprecated browser interface that remains in the repository for now.

Author: ```suggestion
References to Kilo Console and `kilo console` on the architecture pages describe a deprecated browser interface that is deprecated and will be removed in a future release.
```
